test/Microsoft.AspNetCore.Mvc.Core.Test/AcceptedAtActionResultTests.cs

```diff
@@ -24,6 +22,60 @@ namespace Microsoft.AspNetCore.Mvc
 {
     public class AcceptedAtActionResultTests
     {
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData("actionName")]
+        public void Constructor_InitializesActionName(string actionName)
```

Comment:
We generally don't test trivial property \ field assignments in our tests. Is there a reason you thought these were necessary?